src/main/resources/resources/ApplicationResources_pt.properties

```diff
@@ -1111,7 +1111,7 @@ error.studentCurricularPlan.is.not.active.or.concluded = O plano curricular n
 error.studentCurriculum.CurriculumGroup.duplicate.courseGroup = N�o � poss�vel inscrever-se novamente ao grupo '{0}'
 error.studentCurriculum.CurriculumGroup.notEmptyCurriculumGroupModules = N�o � poss�vel remover a inscri��o no grupo '{0}' porque tem inscri��es que dependem dele
 error.studentCurriculum.CycleCurriculumGroup.degree.type.requires.this.cycle.to.exist = N�o � poss�vel remover a inscri��o no grupo '{0}'
-error.studentCurriculum.ExternalCurriculumGroup.cycle.course.group.does.not.belong.to.afinity.of.source = O ciclo
+error.studentCurriculum.ExternalCurriculumGroup.cycle.course.group.does.not.belong.to.afinity.of.source = O ciclo subsequente n�o pertence � lista de afinidades
```

Comment:
The JIRA issue only mentions a bad translation, this seems to go a lot further than that...